### PR TITLE
Run with tini

### PR DESCRIPTION
**Pull request by stanlyDoge**
Signed-off-by: Stanislav Knot <sknot@redhat.com>

Run docker image of bridge with `tini`.

Fixes https://github.com/strimzi/strimzi-kafka-bridge/issues/411

### CODE REVIEW

**Comment by ppatierno** [comment]
Yeah, I agree with Jakub. For example, the Strimzi Kafka, Zookeeper, Connect run scripts use `tini` for starting the "built-in" scripts provided by upstream Kafka for starting (i.e. `kafka-server-start.sh`, `zookeeper-server-start.sh`, ...). It would be the same here.

File: bin/kafka_bridge_run.sh [hunk 1]
```diff
@@ -12,4 +12,4 @@ fi
 # Make sure that we use /dev/urandom
 JAVA_OPTS="${JAVA_OPTS} -Dvertx.cacheDirBase=/tmp -Djava.security.egd=file:/dev/./urandom"
 
-exec java $JAVA_OPTS $KAFKA_BRIDGE_LOG4J_OPTS -classpath "${MYPATH}/../libs/*" io.strimzi.kafka.bridge.Application "$@"
\ No newline at end of file
+java $JAVA_OPTS $KAFKA_BRIDGE_LOG4J_OPTS -classpath "${MYPATH}/../libs/*" io.strimzi.kafka.bridge.Application "$@"
```
Comment (ppatierno) [inline bin/kafka_bridge_run.sh:15]: I think you can leave the `exec` as it was.

File: bin/docker/kafka_bridge_run.sh [hunk 2]
```diff
@@ -56,4 +56,4 @@ function get_gc_opts {
 export JAVA_OPTS="${JAVA_OPTS} $(get_gc_opts)"
 
 # starting Kafka Bridge with final configuration
-${MYPATH}/../kafka_bridge_run.sh --config-file=/tmp/kafka-bridge.properties "$@"
+exec /usr/bin/tini -s -w -e 143 -- ${MYPATH}/../kafka_bridge_run.sh --config-file=/tmp/kafka-bridge.properties "$@"
```
Comment (scholzj) [inline bin/docker/kafka_bridge_run.sh:59]: I think shellcheck would tell you to use quotes around the $MY_PATH variable.

**Comment by ppatierno** [comment]
@stanlyDoge can you run the bridge in Kubernetes and providing the output of `ps -ef` command on the bridge container please?

**Comment by stanlyDoge** [comment]
```
bash-4.2$ ps -ef
UID        PID  PPID  C STIME TTY          TIME CMD
1000160+     1     0  0 07:14 ?        00:00:00 /usr/bin/tini -s -w -e 143 -- /opt/str
1000160+    35     1 35 07:14 ?        00:00:03 java -Dvertx.cacheDirBase=/tmp -Djava.
1000160+    53     0  1 07:14 ?        00:00:00 bash
1000160+    62    53  0 07:14 ?        00:00:00 ps -ef
```

**Pull request merged**